plugins/services/src/js/reducers/JSONSingleContainerParser.js

```diff
@@ -49,6 +49,5 @@ module.exports = [
   portMappings, // Note: must come after portDefinitions, as it uses its information!
   requirePorts,
   residency,
-  fetch,
```

Comment:
I bet it sneaked in when I sorted them and then merged/rebased not careful enough...